dynawo/3rdParty/CMakeLists.txt

```diff
@@ -508,7 +526,14 @@ else()
 
       INSTALL_COMMAND     ""
     )
+
   else()
+    if(CXX11_ENABLED)
+      set(TmpVar "--with-libraries=date_time,filesystem,program_options,serialization,system,log,iostreams,log_setup")
```

Comment:
log_setup is not a valide library name and test is missing

Reply:
It's coming from the original file I think. Anyway it's gone now.

Reply:
And unit_test_framework add :)